static/app/utils/discover/eventView.tsx

```diff
@@ -1173,7 +1173,7 @@ class EventView {
 
   getQueryWithAdditionalConditions() {
     const {query} = this;
-    if (!this.additionalConditions) {
+    if (this.additionalConditions.isEmpty()) {
```

Comment:
This was causing transformations on the query string in discover unnecessarily.

Reply:
Nice 👍